src/Mount.cpp

```diff
@@ -2087,6 +2390,26 @@ void Mount::loop() {
   }
   #endif
 
+  #if (FOCUS_STEPPER_TYPE != STEPPER_TYPE_NONE)
+    // LOGV2(DEBUG_MOUNT, F("Mount: Focuser running:  %d"), _stepperFocus->isRunning());
+
+    if(_stepperFocus->isRunning())
+    {
+      LOGV2(DEBUG_MOUNT, F("Mount: Focuser running at speed %f"), _stepperFocus->speed());
+      _lastFocusMovementTimestamp = now;
+    }
+    else if (_focuserWasRunning)
```

Comment:
`_focuserWasRunning` is always `false`, this code will never run

Reply:
Oops, good catch, fixed.